datafusion/physical-expr/src/aggregate/groups_accumulator/prim_op.rs

```diff
@@ -85,7 +95,7 @@ where
         let values = values[0].as_primitive::<T>();
 
         // update values
-        self.values.resize(total_num_groups, T::default_value());
```

Comment:
previously this always started at zero, now it starts at the specified value.

I think I can now reduce the code in min/max accumulator as well